tutorials/webpack/alias.md

```diff
@@ -0,0 +1,63 @@
+Aliasing is webpack's handy way to shave time and keystrokes
+off importing frequently used modules. You will need the
+`path` npm module as it is how you will tell webpack where
```

Comment:
`path` is a Node.js built-in module, not downloaded via npm: https://nodejs.org/api/path.html